dashboards_bundlers/extensionapp.py

```diff
@@ -59,6 +59,11 @@ def start(self):
                     'label': 'PHP Dashboard bundle (.zip)',
                     'module_name': 'dashboards_bundlers.php_download',
                     'group': 'download'
+                },
+                'dashboards_server_upload': {
+                    'label': 'Dashboard on Dashboard Server',
```

Comment:
Also it's a bit unfortunate that the menu item will always appear even if the backend is not configured properly. We might want to improve the bundler API over in jupyter_cms to account for this.